php/libraries/NDB_BVL_Instrument_LINST.class.inc

```diff
@@ -79,10 +79,17 @@ class NDB_BVL_Instrument_LINST extends \NDB_BVL_Instrument
         $this->addBasicDate('Date_taken', 'Date of Administration', $dateOptions);
 
         if (strrpos($this->testName, '_proband') === false) {
-            $this->addScoreColumn(
-                'Candidate_Age',
-                'Candidate Age (Months)'
-            );
+            if ($this->postMortem) {
```

Comment:
Just found a bug in LINST -- Missing negation here, should be
` if ( ! $this->postMortem) { `

Done Correctly: [ Instrument class line 89](https://github.com/aces/Loris/blob/fef5b306936e7b464dcac49e4c5965198333c1a1/php/libraries/NDB_BVL_Instrument.class.inc#L859)
incorrectly: [ LINST line 77 ](https://github.com/aces/Loris/blob/c2e956a80c6fc1d3c9b70f94bf05e3f60539f2f8/php/libraries/NDB_BVL_Instrument_LINST.class.inc#L77)